parser/parser.y

```diff
@@ -13970,6 +14013,30 @@ LinesTerminated:
 		$$ = $3
 	}
 
+NullDef:
```

Comment:
Why not resuse `TextString`?

Reply:
https://github.com/pingcap/tidb/pull/41332/commits/3326462d9e2eddc50cd78258a8c699de6ba8d4d8